app/posts/claim-funding-for-mentors/2024-12-02-updating-the-files-we-send-to-providers-to-assure-claims.md

```diff
@@ -0,0 +1,78 @@
+title: Updating the files we send to providers to assure claims
+description: Details of how we iterated the comma separated value (CSV) file for providers to match their needs
+date: 2024-12-02
+tags:
+  - research
+  - accredited providers
+  - sampling
+related:
+  items:
+  - text: Round 6 accredited provider research findings
+    href: /claim-funding-for-mentors/round-6-accredited-provider-research-findings/
+---
+
+## Introduction
+
+[Round 6 accredited provider research](/claim-funding-for-mentors/round-6-accredited-provider-research-findings/) highlighted that the information provided in the first iteration of the comma separated value (CSV) file was insufficient for them to assure their claims.
+
+## CSV contents
+
+The CSV that we shared with providers contained the following information:
+
+- claim reference
+- school name
+- school unique reference number (URN)
+- claim hours
+
+The CSV included two additional columns so that the providers could assure each claim and add information into each column:
+
+- claim assured (providers to indicate yes/true or no/false)
+- claim not assured reason (providers to indicate a reason if they had marked the claim as not assured)
+
+## Research findings
+
+During round 6 we shared a CSV with the providers for them to review whether the information matched their needs so that they could accurately assure the claims.
+
+The users stated that the current amount of information was insufficient and recommended that the following information be shared:
+
+- school post code
+- individual mentor names for each claim
+- individual claim hours for each mentor, not the overall claim amount
+  
+Providing this information would enable accredited providers to accurately assure the claims.
+
+## Updated CSV for providers
+
+As a result of the user research, we updated the CSV to include all the following information:
+
+- claim reference
+- school name
+- school unique reference number (URN)
+- school post code
+- individual mentor names for each claim
+- individual claim hours for each mentor, not the overall claim amount
+
+We removed:
+
+- the total claim hours
+
+For each individual mentor, rather than total claim, providers could input information into the two columns:
+
+- claim assured (providers to indicate yes/true or no/false)
+- claim not assured reason (providers to indicate a reason if they had marked the claim as not assured)
+
+## User needs
+
+As a result of updating the CSV we hypothesized that we were able to meet the following user need:
+
+{% from "user-need/macro.njk" import appUserNeed %}
+{{ appUserNeed({
+description: "As a provider,
+I need data related to a claim to match the data that we collect,
+So that we can easily and correctly check the claims against our records when sampling."
+}) }}
+
+## Next steps
+
+In the next round of provider user research we will share the CSV and validate whether the above user need had now been met, so they could successfully assure their claims.
```

Comment:
```suggestion
In the next round of provider user research, we will share the CSV and validate whether the above user need has been met so that they can successfully assure their claims.
```